api.yaml

```diff
@@ -172,6 +173,10 @@ components:
           type: array
           items:
             type: string
+        relatedResources:
```

Comment:
would it be more beneficial to have the `relatedResources` object at the resourceId-level, in `CloudAssetChanges`? If there are multiple `CloudAssetChanges` then the `relatedResources` array would be the same in each, repeated.

Reply:
Could you get a little more in to this? I had planned it to be similar to the other information here, where relations could be added or deleted. And I guess for an understanding check, am I right in thinking that the `CloudAssetChange` array is currently a max of two changesets, grouped by `changeType`?

For the ENI use only one of those comes through at a time so the implementation is pretty simple. But for a theoretical example like an update to a VPC where multiple things could get added or deleted I would expect the two changesets to have different lists of relatedResources that get appended to as the config is worked through. I'm basing this off of my reading of the EC2 update logic though, which might be incomplete.

Reply:
Not sure about the max of two but that makes sense to me. I get it now, I was originally thinking about the implication of tying relatedResources directly to the IP address changes but now I see how they can be unrelated depending on the use case. And I didn't even think about storing the changeType (added, deleted) to the relatedResource event itself and that makes way more sense!